tests/spcs/test_common.py

```diff
@@ -0,0 +1,33 @@
+from snowflake.cli.plugins.spcs.common import validate_and_set_instances
+from tests.testing_utils.fixtures import *
+from click import ClickException
+
+
+@pytest.mark.parametrize(
+    "min_instances, max_instances, expected_max",
+    [
+        (2, None, 2),  # max_instances is None, set max_instances to min_instances
```

Comment:
can we add more test cases:
1. min=None, max=None 
2. min=value, max=None
3. min=None, max=value
4. min=value, max=value